utilities/dotcms/index.js

```diff
@@ -94,38 +98,51 @@ const getToken = ({ user, password, expirationDays, host }) => {
 };
 
 /*
-* Determines if the path ends with /index but only when the / is preceded by a word
-*
-* This is needed to create the right paths for Next.js getStaticPaths' paths array
-* `/destinations/index` becomes `/destinations`
-*
-* @param {string} str - the path (e.g /destinations/index)  
-*/
+ * Determines if the path ends with /index but only when the / is preceded by a word
+ *
+ * This is needed to create the right paths for Next.js getStaticPaths' paths array
+ * `/destinations/index` becomes `/destinations`
+ *
+ * @param {string} str - the path (e.g /destinations/index)
+ */
 const pathEndsWithIndex = (str) => {
     const r = /(?<=\w)(\/index)/;
-    return r.test(str)
-} 
+    return r.test(str);
+};
+
+const setPathsUrls = (arr, url) => {
```

Comment:
This is not a `set` is a `get` the function is returning something

Reply:
The `arr` param name is too generic, doesn't give you any clues of you what it is.

Reply:
https://github.com/dotCMS/dotcms-spa/pull/88/commits/7f4c109db7c501ed79dac6883a133655f4b78692